boards/silabs/radio_boards/xg24_rb4187c/xg24_rb4187c_defconfig

```diff
@@ -8,3 +8,6 @@ CONFIG_GPIO=y
 CONFIG_SOC_GECKO_EMU_DCDC=y
 CONFIG_SOC_GECKO_EMU_DCDC_MODE_ON=y
 CONFIG_PINCTRL=y
+
+# will be deprecated in turn of CONFIG_BT_SEND_ECC_EMULATION##, PR #79931
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
I don't think this is the right place, since it would result in build errors if you try to build a non-Bluetooth application (`BT_TINYCRYPT` depends on `BT`, but `BT` is not enabled). Normally this is solved through `Kconfig.defconfig` in the board directory. For this board there's already an `if BT` section in it, so that would be the appropriate place to add something like:
```
config BT_TINYCRYPT_ECC
	default y
```

That said, it's a bit inconsistent that while you're modifying the uart-pipe for lots of boards you're only modifying this default value for a single board.

Reply:
Good point, 
I'll update other boards too.